guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,45 @@
+[id="viewing-multiple-content-view-environment-hosts"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view hosts with multiple content view environments, or multi-environment hosts, in the {ProjectWebUI} and Hammer CLI.
+
+.Prerequisites
+Ensure that you have at least one multi-environment host. 
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.
++
+[NOTE]
+====
+If you have the *Show new host overview page* setting enabled, the new UI appears by default.
+If you do not have the setting enabled, the classic UI appears by default and you must click the *New UI* button.
+====
++
+. Click *Manage columns*.
+. Under *Content*, click the *Content view environments* checkbox and click *Save*.
+. Content view environments for each host are displayed under the *Content view environments* column.
+. Click on the content view environments to display details.
+
+On the *Host Details* page, the *Content view environments* card displays the details of the content view environments.
+
+[NOTE]
+====
+If you reassign the content view environments of a multi-environment host from the *Content view environments* card, the host's content view environments are overwritten and it is no longer a multi-environment host. 
+====
+
+.CLI procedure
+. Run the following command to view a list of hosts in your organization:
```

Comment:
```suggestion
. View a list of hosts in your organization:
```

suggestion: reword

Avoid using "the following command" because it does not add anything useful to the docs. The sentence ends with a colon and code block which makes it obvious that users have to run this. The sentence also uses imperative style AFAICT.